### PR TITLE
Add first 2 french translations

### PR DESCRIPTION
**Pull request by minscof**
The first 2 french translations with git to check the process

### CODE REVIEW

**Comment by mherger** [comment]
Perfect! The way to go. I'm looking forward to seeing the big commit :-). Thanks so much!

**Comment by minscof** [comment]
I had translated all the server strings.txt and check on my local squeezebox server. I think it is ok for this file.

File: strings.txt [hunk 1]
```diff
@@ -7638,7 +7640,7 @@ SETUP_NOGENREFILTER
 	EN	Filter Genres at the Album & Track Level.
 	ES	Filtrar géneros por álbum y pista.
 	FI	Suodata genret levy- ja raitatasolla.
-	FR	Filtrer les genres au niveau des albums et des morceaux
+	FR	Portée du filtre surr les genres
```
Comment (mherger) [inline strings.txt:7643]: "surr"? Typo?

File: strings.txt [hunk 2]
```diff
@@ -7621,7 +7623,7 @@ SETUP_SEARCHSUBSTRING_DESC
...
-	FR	Lors d'une recherche parmi les fichiers multimédias, le serveur peut afficher comme résultat les éléments contenant les mots spécifiés ou seulement les éléments dont le nom débute par les mots spécifiés. Par exemple, une recherche sur RAIN affichera les morceaux contenant le mot RAINBOW si vous sélectionnez la recherche par début de mot. Si vous sélectionnez l'option de recherche à l'intérieur des mots, TRAIN sera également renvoyé comme résultat.
+	FR	Lors d'une recherche dans la bibliothèque multimédia, les résultats peuvent correspondre à des mots contenant les termes spécifiés ou seulement commençant par les termes spécifiés. Par exemple, une recherche sur RAIN affichera les morceaux contenant le mot RAINBOW si vous sélectionnez la recherche en début de mot. Si vous sélectionnez l'option de recherche n'importe oùs, TRAIN sera également renvoyé comme résultat.
```
Comment (mherger) [inline strings.txt:7626]: "n'importe oùS"? Typo?

File: strings.txt [hunk 3]
```diff
@@ -12847,7 +12864,7 @@ BROWSE_MUSIC
 	EN	Music Library
 	ES	Biblioteca de música
 	FI	Musiikkikirjasto
-	FR	Bibliothèque musicale
+	FR	Bibliothèque multimédia
```
Comment (mherger) [inline strings.txt:12867]: Are you sure about this change? It's the menu item shown on the players etc., where you can only access music files, but none of the other media types (movies, photos).

**Comment by mherger** [comment]
Thanks a lot for all your work!
I'm a bit hesitant about the "bibliothèque musicale" vs. "bibliothèque multimédia" change. This is a term which has been used this way for years, and it's central in all our UIs. I don't think it's wrong. After all it's music only. I changed my phone to French for a test. And eg. Spotify as well as Apple's Music app would present the menus as "Ma Musique".

**Comment by minscof** [comment]
I hesitated before replacing "bibliothèque musicale" by "bibliothèque multimédia". I m going to keep "bibliothèque musicale" and make others corrections.

How can I extract only the french translation to read it and detect some mistakes ? (strings.txt is a very big file, not so easy to use).

**Comment by minscof** [comment]
I think it is ok now for this file.

**Comment by mherger** [comment]
Now you changed all "bibliothèques multimédias" to "... musicales", even in places where multimedia is correct (eg. the UPnP server _can_ server images and vidéos). Please compare your branch to the origin and make sure none of those strings got changed.  Thanks!

**Comment by mherger** [comment]
@minscof - I've merged a first chunk of changes. I did _not_ merge the "musical/multimédia" changes. Please revert them all.

One more thing: please put your translations in alphabetical order of the language. In most cases this is right after EN for the missing strings.

You should now probably to a merge from "remote" to sync up your fork. Thanks a lot!

**Pull request closed without merging**